k8s_migrate_job.yml

```diff
@@ -16,6 +16,7 @@ spec:
         - name: app
           image: 754256621582.dkr.ecr.eu-west-2.amazonaws.com/book-a-secure-move/hmpps-book-secure-move-api:__ENV__.latest
```

Comment:
We probably don't want to use `:__ENV__.latest` here anyway. 
It's more robust to use the actual docker tag for the build, but we might want to address the deployment job at the same time for that.